I.write-tests/03-remove-vowels-from-array/remove-vowels-in-array.test.js

```diff
@@ -1,11 +1,15 @@
-test('remove vowels from all words in array', function () {
-    // Arrange
+const removeVowelsForWords = require("./remove-vowels-in-array");
 
-    // Act
-
-    // Assert
-})
+test("remove vowels from all words in array", function() {
+  // Arrange
+  const people = ["Irina", "Etza", "Daniel"];
```

Comment:
What would happen if we add `Ashley` to the array?
If our function is supposed to remove vowels, then the result should be `shl`.
Can we confirm if that actually work and, if not, fix the bug?

Reply:
I did test  and it works and pass the test